src/maps/OverlayedMap.cpp

```diff
@@ -481,6 +512,33 @@ void OverlayedMap::drawFix(const xdata::Fix& fix, double scale) {
     }
 }
 
+void OverlayedMap::drawScale(double nmPerPixel) {
+    bool useFeet = (nmPerPixel < 0.005);
+    std::string units = useFeet ? "ft" : "nm";
+    double perPixel = useFeet ? (nmPerPixel * 6076) : nmPerPixel;
+    double maxRange = 300 * perPixel;
+    double step = std::pow(10, ((int)std::log10(maxRange)));

```

Comment:
Is the cast to `int` done to truncate the decimal part? In that case, please use `std::floor` instead to make it obvious.